hedera-node/hedera-app/src/xtest/java/contract/ScaffoldingComponent.java

```diff
@@ -21,6 +21,7 @@
 import com.hedera.node.app.spi.workflows.HandleContext;
 import com.hedera.node.app.spi.workflows.TransactionHandler;
 import com.hedera.node.app.state.HederaState;
+import com.hedera.node.app.state.WorkingStateAccessor;
```

Comment:
(what's a "working" state as opposed to all the other kinds of states?  the javadoc comment at `WorkingStateAccessor` doesn't define it. is it different than the "current" state at this transaction?)

Reply:
No, that's right..."working state" = "current state" = "mutable state".